cmd/waku/flags.go

```diff
@@ -171,13 +171,13 @@ var (
 		Destination: &options.CircuitRelay,
 		EnvVars:     []string{"WAKUNODE2_CIRCUIT_RELAY"},
 	})
-	ForceUnreachable = altsrc.NewBoolFlag(&cli.BoolFlag{
-		Name:        "unreachable",
-		Usage:       "Force the node to be unreachable. WARNING: This flag is created for testing circuit relay and is not meant to be used in production",
-		Value:       false,
+	ForceReachability = altsrc.NewStringFlag(&cli.StringFlag{
+		Name:        "reachability",
```

Comment:
Better to name the flag also to be force-reachability (which gives a vibe for users not to use it).